public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -97,7 +93,6 @@ export class PrometheusDatasource
   access: 'direct' | 'proxy';
   basicAuth: any;
   withCredentials: any;
-  metricsNameCache = new LRUCache<string, string[]>({ max: 10 });
```

Comment:
@gtk-grafana this wasn't in use so I removed it.

Reply:
Yeah good, call there should also be a `labelsCache` and `labelValuesCache` which can be removed in the language_provider.ts as well